aws-apprunner/README.md

```diff
@@ -142,7 +142,7 @@ pvault --addr <VAULT URL from above> --authtoken '<token from the secret manager
 | <a name="input_pvault_port"></a> [pvault\_port](#input\_pvault\_port) | Pvault application port number | `string` | `"8123"` | no |
 | <a name="input_pvault_repository"></a> [pvault\_repository](#input\_pvault\_repository) | Pvault repository public image | `string` | `"public.ecr.aws/s4s5s6q8/pvault-server"` | no |
 | <a name="input_pvault_service_license"></a> [pvault\_service\_license](#input\_pvault\_service\_license) | Pvault license code https://piiano.com/docs/guides/install/pre-built-docker-containers. Cannot be set if var.create\_secret\_license is set to 'true' | `string` | `""` | no |
-| <a name="input_pvault_tag"></a> [pvault\_tag](#input\_pvault\_tag) | n/a | `string` | `"1.6.0"` | no |
+| <a name="input_pvault_tag"></a> [pvault\_tag](#input\_pvault\_tag) | Pvault version (example) | `string` | `"1.6.1"` | no |
```

Comment:
Cant we replace to real version here?

Reply:
It will be harder as I am afraid the regex will find wrong things. Let's keep it as "example" for now